pset-functions.js

```diff
@@ -31,6 +31,31 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack(playerCardScore, dealerCardScore) {
+  const cards = [playerCardScore, dealerCardScore]
+  const less21 = cards.filter(cards => {
+                    return cards <= 21
+                  })
+  const max = Math.max(...less21)
```

Comment:
@jacobkelman cool use of the spread operator combined with `Math.max()` to get the highest of the two values